iml-system-test-utils/src/lib.rs

```diff
@@ -273,14 +273,22 @@ impl Config {
     pub fn get_setup_config(&self) -> String {
         match &self.test_type {
             TestType::Rpm => format!(
-                r#"USE_STRATAGEM = {}
-BRANDING = "{}""#,
+                r#"USE_STRATAGEM={}
+BRANDING={}
+LOG_LEVEL=10
```

Comment:
This makes me thinking what the 10 is. I was expecting `LOG_LEVEL` to be usual one of `trace`, `debug`, `warn` etc.
How is it used?

Reply:
```
$ python
>>> import logging
>>> logging.DEBUG
10
```

Reply:
`LOG_LEVEL` is the python's log level and it's an integer. Here is the python repl output:
```
>>> import logging
>>> logging.DEBUG
10
>>> logging.INFO
20
```